docs/guide/controller/setup.md

```diff
@@ -16,14 +16,19 @@ An example policy with the minimum rights can be found at [iam-policy.json](../.
 ## Installation
 You can choose to install ALB ingress controller via Helm or Kubectl
 ### Helm
-1. Install [Helm App Registry plugin](https://coreos.com/apps)
+1. Install Helm registry
+    ```bash
+    helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+    ```
 
 2. Install ALB ingress controller
 
     ``` bash
-    helm registry install quay.io/coreos/alb-ingress-controller-helm
+    helm install incubator/aws-alb-ingress-controller
```

Comment:
I think it's better to use `helm install incubator/aws-alb-ingress-controller --set autoDiscoverAwsRegion=true --set autoDiscoverAwsVpcID=true --set clusterName=MyClusterName`,  which makes the required parameters explicit, otherwise, people will get an controller installed but not working. (I should make autoDiscoverAwsRegion, autoDiscoverAwsVpcID default to false, just to pass helm's e2e testing running in GKE cluster 😶 ) Feels sorry for the end user experience though...